db/seeds.rb

```diff
@@ -51,7 +51,7 @@
   },
   {
     application: "LTI Starter App",
-    tenant: "starter-app",
+    tenant: Rails.application.secrets.default_lti_key,
```

Comment:
Unfortunately this is tricky because of how we develop our applications. Locally we use atomicjolt.xyz and every subdomain has to be setup by our bootstrap script which gets the subdomains from the .env file (it can't read ruby code). For the sake of a quick example we seed a basic LTI application which is called lti-example. We have to get the bootstrap script to generate an nginx config and get the seeds to add a db entry with the matching domain. The only single point to store that data is in the .env. We could skip the secrets and put ENV['something'] in the seeds. Would that make it simpler?